resources/views/structure/navbar.blade.php

```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-expand-md navbar-bg-color white-text fixed-top">
+    <a class="navbar-brand" href="{{url('/')}}">IMDB Touch</a>
+    <button class="navbar-toggler"
+            type="button"
+            data-toggle="collapse"
+            data-target="#navbarsExampleDefault"
+            aria-controls="navbarsExampleDefault"
+            aria-expanded="false"
+            aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+        <ul class="navbar-nav mr-auto">
+            <li class="nav-item active">
+                <a class="nav-link" href="{{url('/')}}">Home <span class="sr-only">(current)</span></a>
+            </li>
+        </ul>
+        {!! Form::open(['route' => 'movie.search', 'method' => 'POST', 'class' => 'form-inline my-2 my-lg-0']) !!}
```

Comment:
Given you have only one form and search field in the app don't you think using an additional lib for handling form related stuff is kind of an overhead?

Reply:
Sure it is, on this I have nothing to say to justify this. LOL